### ISSUE TITLE
STM32Cube7 v1.17.1 (Patch)

### ISSUE THREAD

**Issue by SteveDSP** [STMicroelectronics/STM32CubeF7#100]
**Caution**
The Issues are strictly limited for the reporting of problem encountered with the software provided in this project.
For any other problem related to the STM32 product, the performance, the hardware characteristics and boards, the tools the environment in general, please post a topic in the [ST Community/STM32 MCUs forum](https://community.st.com/s/group/0F90X000000AXsASAW/stm32-mcus).

**Describe the set-up**

STM32F769I-DISCO

STM32CubeIDE - 1.13.1

**Describe the bug**

After updating with the FW Patch v1.17.1, my project now complains"Code Generation could not be done probably because the necessary firmware probably isn't........"

When creating the project, STM32, it's possible to click next and under 'Firmware Package Name and Version', there is a drop down. If I select v1.17.1, the error above is still generated. When you proceed further with the auto code generation from the IOC, none of source is included and the project build icon is greyed out. 

**How To Reproduce**
1. As above, install v1.17.1 FW7 Patch and follow instructions above.

2. When looking at this, I found that the ZIP file for the FW7 v1.17.1 unzips with the same name as the release, v1.17.0. If I select v1.17.0 as outlined above after unzipping and  copying the new update, the "Code Generation could not be done.." pop up still displays.
3. The only workaround for me is to unzip my original v1.17.0 package into the repository, and select v1.17.0, the project is then created successfully and following auto code generation compiles and runs.

In Summary:

 If I use the 'patched version' v1.17.1, (renamed or not once unzipped) whatever I choose in "Firmware Package Name and Version" does not help.
 If I use my original version v1.17.0, and select this in "Firmware Package Name and Version", all is well. 

**Additional context**

Perhaps there is a mixup in version/patch naming with regards to the repository somewhere. I find it odd that the zip opens with the same name as the original version.

**Screenshots**

<img width="705" alt="Screenshot 2023-08-22 at 16 53 04" src="https://github.com/STMicroelectronics/STM32CubeF7/assets/38793034/ed4177d1-cae8-4048-b4fd-5302466d95a4">

**Comment by ASELSTM** [comment]
Hi @SteveDSP,

The point you reported is actually not related to the firmware published in this repository. Unfortunately, we don't treat aspect related to ecosystem in our GitHub repositories. They are rather treated in the [ST Community](https://community.st.com/) .

Since this issue is not directly related to the STM32Cube firmware, please allow me then to close this thread. Thank you for your comprehension.

With regards,
